### PR TITLE
add vulnerability report policy

### PR DESCRIPTION
**Pull request by aead**
## Description
add vulnerability report policy

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#414b73bf-7549-4df4-80cc-0aee064fd19d) |
<a name="414b73bf-7549-4df4-80cc-0aee064fd19d"></a>
### 11084-c2baf79/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c2.minio.io:31948
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 85b0f1921f3f:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 2 minutes and 40 seconds
(4/15) Running aws-sdk-ruby tests ... done in 34 seconds
(5/15) Running awscli tests ... FAILED in 3 seconds
{
  "name": "awscli",
  "duration": 1497,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}
(5/15) Running healthcheck tests ... done in 1 seconds
(6/15) Running mc tests ... done in 3 minutes and 37 seconds
(7/15) Running minio-dotnet tests ... done in 1 minutes and 39 seconds
(8/15) Running minio-go tests ... FAILED in 1 minutes and 45 seconds
{
  "args": {
    "bucketName": "minio-go-test-15dch6zpco1znn91",
    "objectName": "test-object",
    "opts": "",
    "size": -1
  },
  "duration": 1187,
  "function": "PutObject(bucketName, objectName, reader,size,opts)",
  "message": "Unexpected size",
  "name": "minio-go: testPutObjectStreaming",
  "status": "FAIL"
}
(8/15) Running minio-java tests ... FAILED in 10 minutes and 2 seconds
{
  "name": "minio-java",
  "function": "putObject()",
  "args": "[user metadata]",
  "duration": 173,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = AuthenticationFailed, message = -> github.com/Azure/azure-storage-blob-go/azblob.newStorageError, github.com/Azure/azure-storage-blob-go@v0.10.0/azblob/zc_storage_error.go:42\n===== RESPONSE ERROR (ServiceCode=AuthenticationFailed) =====\nDescription=Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\nRequestId:1c8f309c-501e-012b-2ac4-cfdf03000000\nTime:2020-12-11T13:51:39.3035239Z, Details: \n   AuthenticationErrorDetail: The MAC signature found in the HTTP request 'yolCZkJIGbls8VvIdCjYOJ3xBInJDLTfhzz3k1qeBlU=' is not the same as any computed signature. Server used following string to sign: 'PUT\n\n\n128\n\napplication/xml\n\n\n\n\n\n\nx-ms-blob-cache-control:\nx-ms-blob-content-disposition:\nx-ms-blob-content-encoding:\nx-ms-blob-content-language:\nx-ms-blob-content-type:application/octet-stream\nx-ms-client-request-id:9bf1b4c6-12ff-4fe5-757f-2b8a255f8de7\nx-ms-date:Fri, 11 Dec 2020 13:51:39 GMT\nx-ms-meta-my_header1:a   b   c\nx-ms-meta-my_header2:\"a   b   c\"\nx-ms-meta-my_project:Project One\nx-ms-meta-my_unicode_tag:å•†å“\nx-ms-version:2019-02-02\n/minioazure/minio-java-test-1bi2g1s/minio-java-test-3oipcm3\ncomp:blocklist\ntimeout:1501'.\n   Code: AuthenticationFailed\n   PUT https://minioazure.blob.core.windows.net/minio-java-test-1bi2g1s/minio-java-test-3oipcm3?comp=blocklist&timeout=1501\n   Authorization: REDACTED\n   Content-Length: [128]\n   Content-Type: [application/xml]\n   User-Agent: [APN/1.0 MinIO/1.0 MinIO/2020-12-11T13:25:04Z]\n   X-Ms-Blob-Cache-Control: []\n   X-Ms-Blob-Content-Disposition: []\n   X-Ms-Blob-Content-Encoding: []\n   X-Ms-Blob-Content-Language: []\n   X-Ms-Blob-Content-Type: [application/octet-stream]\n   X-Ms-Client-Request-Id: [9bf1b4c6-12ff-4fe5-757f-2b8a255f8de7]\n   X-Ms-Date: [Fri, 11 Dec 2020 13:51:39 GMT]\n   X-Ms-Meta-My_header1: [a   b   c]\n   X-Ms-Meta-My_header2: [\"a   b   c\"]\n   X-Ms-Meta-My_project: [Project One]\n   X-Ms-Meta-My_unicode_tag: [商品]\n   X-Ms-Version: [2019-02-02]\n   --------------------------------------------------------------------------------\n   RESPONSE Status: 403 Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\n   Content-Length: [1092]\n   Content-Type: [application/xml]\n   Date: [Fri, 11 Dec 2020 13:51:38 GMT]\n   Server: [Microsoft-HTTPAPI/2.0]\n   X-Ms-Error-Code: [AuthenticationFailed]\n   X-Ms-Request-Id: [1c8f309c-501e-012b-2ac4-cfdf03000000]\n\n\n, bucketName = minio-java-test-1bi2g1s, objectName = minio-java-test-3oipcm3, resource = /minio-java-test-1bi2g1s/minio-java-test-3oipcm3, requestId = 164FADCF9C4F1C99, hostId = c5509c5e-42a9-4172-8c94-f32b943e6df4)\nrequest={method=PUT, url=http://minio-c2.minio.io:31948/minio-java-test-1bi2g1s/minio-java-test-3oipcm3, headers=x-amz-meta-My-Unicode-Tag: 商品\nx-amz-meta-My-Project: Project One\nx-amz-meta-My-header1: a   b   c\nx-amz-meta-My-Header2: \"a   b   c\"\nContent-Type: application/octet-stream\nHost: minio-c2.minio.io:31948\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.0.3\nContent-MD5: A9oFTxee7YVcJ9fWsgQeKg==\nx-amz-content-sha256: 1ff7959f86334ddc5c188a5083268f600146328b2b6c5185e75bf7d9387d6b74\nx-amz-date: 20201211T135139Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20201211/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date;x-amz-meta-my-header1;x-amz-meta-my-header2;x-amz-meta-my-project;x-amz-meta-my-unicode-tag, Signature=*REDACTED*\n}\nresponse={code=403, headers=Accept-Ranges: bytes\nContent-Length: 3086\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO\nVary: Origin\nX-Amz-Request-Id: 164FADCF9C4F1C99\nX-Xss-Protection: 1; mode=block\nDate: Fri, 11 Dec 2020 13:51:39 GMT\n}\n >>> [io.minio.MinioClient.execute(MinioClient.java:775), io.minio.MinioClient.putObject(MinioClient.java:4547), io.minio.MinioClient.putObject(MinioClient.java:2713), io.minio.MinioClient.putObject(MinioClient.java:2830), FunctionalTest.testPutObject(FunctionalTest.java:763), FunctionalTest.putObject(FunctionalTest.java:890), FunctionalTest.runObjectTests(FunctionalTest.java:3751), FunctionalTest.runTests(FunctionalTest.java:3783), FunctionalTest.main(FunctionalTest.java:3927)]"
}
(8/15) Running minio-js tests ... FAILED in 46 seconds
{
  "name": "minio-js",
  "function": "\"after all\" hook in \"functional tests\"",
  "duration": 78,
  "status": "FAIL",
  "error": "S3Error: The bucket you tried to delete is not empty at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:68:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:156:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:80:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:684:10) at endReadableNT (_stream_readable.js:1327:12) at processTicksAndRejections (internal/process/task_queues.js:80:21)"
}
(8/15) Running minio-py tests ... done in 18 minutes and 25 seconds
(9/15) Running s3cmd tests ... done in 2 minutes and 12 seconds
(10/15) Running s3select tests ... done in 38 seconds
(11/15) Running security tests ... done in 0 seconds

Executed 11 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

File: VULNERABILITY_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+ - Whether the reported vulnerability exists.
+ - The conditions that are required such that the vulnerability can be exploited.
+ - The steps required to fix the vulnerability.
+
+In general, if the vulnerability exists in one of the MinIO code bases
+itself - not in a code dependency - then MinIO will, if possible, fix
+the vulnerability or implement reasonable countermeasures such that the
+vulnerability cannot be exploited anymore.
+
```
Comment (klauspost) [inline VULNERABILITY_REPORT.md:39]: Maybe add a section on where to send the report?

Comment (aead) [reply VULNERABILITY_REPORT.md:39]: Github has kind of a "standard" for that: https://github.com/minio/minio/security/policy

But we could add a oneliner...

**Review by kannappanr — APPROVED** [review]
LGTM

**Pull request merged**